intellst-api/src/Controller/AuthController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Entity\User;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class AuthController extends AbstractController
+{
+    public function register(Request $request, UserPasswordEncoderInterface $encoder)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $username = $request->request->get('_username');
+        $password = $request->request->get('_password');
+
+        $user = new User($username);
+        $user->setPassword($encoder->encodePassword($user, $password));
+        $em->persist($user);
+        $em->flush();
+
+        return new Response(sprintf('User %s successfully created', $user->getUsername()));
+    }
+
+    public function api()
+    {
+        return new Response(sprintf('Logged in as %s', $this->getUser()->getUsername()));
+    }
+}
```

Comment:
add blank line at the end of the file please. (Consider configuring PHPStorm to use PSR-12 code style for PHP)